independent-publisher-connectors/Microsoft Graph Add Users/apiProperties.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{YOUR_CLIENT_ID}",
+          "scopes": [],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "{YOUR_REDIRECT_URL}",

```

Comment:
Kindly remove redirectUrl, this will be added by the certification team during certification.